gwsumm/data/timeseries.py

```diff
@@ -106,7 +106,7 @@
 SHORT_HOFT_TYPES = {  # map aggregated h(t) type to short h(t) type
     'H1_HOFT_C00': 'H1_DMT_C00',
     'L1_HOFT_C00': 'L1_DMT_C00',
-    'V1Online': 'V1_llhoft',
+    'HoftOnline': 'V1_llhoft',
```

Comment:
I recommend _appending_ the new type rather than _replacing_, so that the code still works for O3 in principle.

```suggestion
    'HoftOnline': 'V1_llhoft',
    'V1Online': 'V1_llhoft',
```